README.md

```diff
@@ -1,18 +1,9 @@
 # ProjectTides
 
 To start your Phoenix server:
```

Comment:
I don't think the steps below are just to start the Phoenix server. Should we rephrase/remove?

Reply:
Updated the readme.